client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
> This is both an untrue and unfair statement. 

No, that's exactly what's happening here.

> said consistency was for reusing the CSS for that element instead of duplicating the code
> how that can be a bad thing

Breaking a good UI for a better code... Are you doing this for users or just for yourself?

> Expecting a user to guess what a blank area that should be inert does when they click on it, as well as the boundaries of the clickable area is not one of those good things.

There's no "guess". It's intuitive: if user miss that symbol for a few times and condensed message still opens as expected - it becomes intuitive and more relaxed way.
Good UI should help user to do things he wants to do, but you break this and go the opposite direction "for consistency and better code". 

> Also, I don't just break things because I like it

C'mon, just admit it :rofl: I have 5 closed PRs because of this.

>  I suggest you look at the code and run it locally, you'll realize this is a non-issue.

I run it since 15.05.2017 and telling you: it's A BIG issue. How long did you run it before changing things?

> that cause the current usability to be negatively affected, we'll obviously resolve this. In time.

Oh, don't worry, that won't be a case. How many users use this PR? When this go to the release most of them won't even know that there was such nice UI, so I don't expect any complains.